plugins/c8y_log_plugin/src/main.rs

```diff
@@ -238,23 +213,26 @@ fn create_init_logs_directories_and_files(
         "tedge",
         "tedge",
         0o755,
```

Comment:
Again not related to your PR, but makes no sense the supported operation file is executable. Also, it will keep consistency with `c8y-configuration-plugin`.

```suggestion
        0o644,
```

Reply:
@PradeepKiruvale @initard It needs to be executable also for inotify? We need to decide either `0o644` or `0o755`.
It's correct that we are mixing up, on one hand, the config plugin creates supported operation files with `0o644`, on the other hand, the log plugin creates the supported operation file with `0o755`.

Reply:
@rina23q I spoke to @initard regarding `0644` permission to `c8y-log-plugin.toml`, and he said it will work.